widget/embedded/src/components/Quote/QuoteTrigger .tsx

```diff
@@ -0,0 +1,185 @@
+import type { QuoteTriggerImagesProps, QuoteTriggerProps } from './Quote.types';
+
+import { i18n } from '@lingui/core';
+import {
+  ChevronDownIcon,
+  ChevronRightIcon,
+  Divider,
+  Image,
+  Tooltip,
+  Typography,
+} from '@rango-dev/ui';
+import React from 'react';
+
+import { getContainer } from '../../utils/common';
+import { getUniqueBlockchains } from '../../utils/quote';
+
+import {
+  IconContainer,
+  ImageContainer,
+  MoreStep,
+  Row,
+  Trigger,
+} from './Quote.styles';
+
+const MAX_STEPS = 4;
+const MAX_BLOCKCHAINS = 5;
```

Comment:
![RNG-Widget – Figma (1)](https://github.com/rango-exchange/rango-client/assets/141495369/77cf7299-02a2-479e-8853-a4b0483c5058)
It seems that we have to display 5 blockchains